cmd/livepeer/livepeer.go

```diff
@@ -901,6 +905,26 @@ func main() {
 		drivers.NodeStorage = drivers.NewMemoryDriver(n.GetServiceURI())
 	}
 
+	if *metadataPublishTimeout > 0 {
+		server.MetadataPublishTimeout = *metadataPublishTimeout
+	}
+	if *metadataQueueUri != "" {
+		uri, err := url.ParseRequestURI(*metadataQueueUri)
+		if err != nil {
+			glog.Fatalf("Error parsing -metadataQueueUri: %q", err)
```

Comment:
I see we log err= for the AMQP error, but not here. Let's do so here as well then.

We started doing the key=value logging format in a few places in the codebase, but not yet in `livepeer.go`, but I'm cool with making the transition in this file as well.